{\rtf1\ansi\ansicpg1252\cocoartf2822
\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\froman\fcharset0 TimesNewRomanPS-BoldMT;\f1\froman\fcharset0 TimesNewRomanPSMT;\f2\froman\fcharset0 TimesNewRomanPS-BoldItalicMT;
\f3\froman\fcharset0 TimesNewRomanPS-ItalicMT;}
{\colortbl;\red255\green255\blue255;\red0\green0\blue0;\red255\green255\blue255;\red0\green0\blue233;
}
{\*\expandedcolortbl;;\cssrgb\c0\c0\c0;\cssrgb\c100000\c100000\c100000;\cssrgb\c0\c0\c93333;
}
{\*\listtable{\list\listtemplateid1\listhybrid{\listlevel\levelnfc0\levelnfcn0\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{decimal\}}{\leveltext\leveltemplateid1\'01\'00;}{\levelnumbers\'01;}\fi-360\li720\lin720 }{\listname ;}\listid1}
{\list\listtemplateid2\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid101\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid2}
{\list\listtemplateid3\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid201\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid3}
{\list\listtemplateid4\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid301\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid4}
{\list\listtemplateid5\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid401\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid5}
{\list\listtemplateid6\listhybrid{\listlevel\levelnfc0\levelnfcn0\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{decimal\}}{\leveltext\leveltemplateid501\'01\'00;}{\levelnumbers\'01;}\fi-360\li720\lin720 }{\listname ;}\listid6}
{\list\listtemplateid7\listhybrid{\listlevel\levelnfc23\levelnfcn23\leveljc0\leveljcn0\levelfollow0\levelstartat1\levelspace360\levelindent0{\*\levelmarker \{disc\}}{\leveltext\leveltemplateid601\'01\uc0\u8226 ;}{\levelnumbers;}\fi-360\li720\lin720 }{\listname ;}\listid7}}
{\*\listoverridetable{\listoverride\listid1\listoverridecount0\ls1}{\listoverride\listid2\listoverridecount0\ls2}{\listoverride\listid3\listoverridecount0\ls3}{\listoverride\listid4\listoverridecount0\ls4}{\listoverride\listid5\listoverridecount0\ls5}{\listoverride\listid6\listoverridecount0\ls6}{\listoverride\listid7\listoverridecount0\ls7}}
\paperw11900\paperh16840\margl1440\margr1440\vieww17120\viewh12920\viewkind0
\deftab720
\pard\pardeftab720\sa321\partightenfactor0

\f0\b\fs36 \cf0 \expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 # Custom Meat Spoilage Gene Database\

\f1\b0 This repository contains the custom-curated reference database for meat spoilage-related bacterial genes, as described in our associated manuscript.\
\pard\pardeftab720\sa240\partightenfactor0
\cf0 # 
\f0\b Overview\

\f1\b0 This database was constructed to improve the functional profiling of metagenomic data from meat processing environments, meat products, and other food-related ecosystems.\
The database is built from two main components:\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa240\partightenfactor0
\ls1\ilvl0
\f0\b \cf0 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	1	}1.
\f1\b0  \expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 A core set of 336 spoilage-related genes curated from an extensive review of scientific literature.\
\ls1\ilvl0
\f0\b {\listtext	2	}2. 
\f1\b0 An expanded set of 4,653 orthologous sequences sourced from the NCBI protein database to enhance detection sensitivity.\
\
\pard\pardeftab720\sa240\partightenfactor0
\cf0 All genes in this database are categorised into eight main functional pathways:\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa240\partightenfactor0
\ls2\ilvl0\cf0 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Protein Metabolism\
\ls2\ilvl0\kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Lipid Metabolism\
\ls2\ilvl0\kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Sulphur Metabolism\
\ls2\ilvl0\kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Regulatory Pathways\
\ls2\ilvl0\kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Siderophore Production\
\ls2\ilvl0\kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Biofilm Formation\
\ls2\ilvl0\kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Stress Adaptation\
\ls2\ilvl0\kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Quorum Sensing\
\pard\tx720\pardeftab720\sa240\partightenfactor0
\cf0 \
\pard\pardeftab720\sa298\partightenfactor0

\f0\b \cf0 # Files in this Repository
\f1\b0 \
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa240\partightenfactor0
\ls3\ilvl0
\f0\b \cf0 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 spoilage_genes_database.fa
\f1\b0 : The FASTA file containing all 4,653 de-duplicated protein sequences (336 core genes + 4,317 orthologs).\
\ls3\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 spoilage_annotations.csv
\f1\b0 : A spreadsheet containing detailed metadata for each sequence in the database.\
\pard\pardeftab720\sa240\partightenfactor0
\cf0 \
\pard\pardeftab720\sa298\partightenfactor0

\f0\b \cf0 # FASTA Header Format (spoilage_genes_database.fa)
\f1\b0 \
\pard\pardeftab720\sa240\partightenfactor0
\cf0 The FASTA headers are formatted in a single, pipe-delimited (|) line for easy parsing.\
\pard\pardeftab720\sa240\partightenfactor0

\f0\b \cf0 * Example:
\f1\b0  >gi|1096123828|gb|APA31905.1|adnA|Flagellar and Biofilm Transcriptional Regulator|Pseudomonas fluorescens\

\f0\b Breakdown:
\f1\b0 \
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa240\partightenfactor0
\ls4\ilvl0
\f0\b \cf0 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Field 1 (e.g., gi|1096123828):
\f1\b0  GenInfo Identifier (gi) from NCBI.\
\ls4\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Field 2 (e.g., gb|APA31905.1):
\f1\b0  GenBank Accession number for the protein.\
\ls4\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Field 3 (e.g., adnA):
\f1\b0  The gene symbol.\
\ls4\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Field 4 (e.g., Flagellar \outl0\strokewidth0 and Biofilm Transcriptional Regulator\outl0\strokewidth0 \strokec2 ):
\f1\b0  The functional description of the protein.\
\ls4\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Field 5 (e.g., 
\f2\i Pseudomonas fluorescens
\f0\i0 ):
\f1\b0  The source organism name.\
\pard\pardeftab720\sa240\partightenfactor0
\cf0 \
\pard\pardeftab720\sa298\partightenfactor0

\f0\b \cf0 # Annotation File Legend (spoilage_annotations.csv)
\f1\b0 \
\pard\pardeftab720\sa240\partightenfactor0
\cf0 The annotation file contains the following columns:\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa240\partightenfactor0
\ls5\ilvl0
\f0\b \cf0 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Species
\f1\b0 : The species name as described in the source article.\
\ls5\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Function_Category
\f1\b0 : The broad metabolic pathway the gene belongs to (e.g., Quorum Sensing, Biofilm Formation, etc.).\
\ls5\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Gene_Symbol
\f1\b0 : The common name or symbol of the gene (e.g., 
\f3\i luxS
\f1\i0 , 
\f3\i pfaA
\f1\i0 ).\
\ls5\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Functional_Code_from_NCBI
\f1\b0 : The specific metabolic product or function of the gene as described in the NCBI database.\
\ls5\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Encoded_protein_from_articles
\f1\b0 : The specific product or function as described in the original publication.\
\ls5\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Cellular_Component
\f1\b0 : The specific location of the gene product (e.g., "Cell membrane"), sourced from UniProt.\
\ls5\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Molecular_Function
\f1\b0 : The specific molecular activity of the gene product (e.g., "ATP binding"), sourced from UniProt.\
\ls5\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Biological_Process
\f1\b0 : The biological process the gene is involved in (e.g., "siderophore biosynthetic process"), sourced from UniProt.\
\pard\pardeftab720\sa240\partightenfactor0
\cf0 \
\pard\pardeftab720\sa298\partightenfactor0

\f0\b \cf0 # Citation
\f1\b0 \
\pard\pardeftab720\sa240\partightenfactor0
\cf0 If you use this database in your research, please cite both this data repository and our associated manuscript.\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa240\partightenfactor0
\ls6\ilvl0
\f0\b \cf0 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	1	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Data Repository (Cite this):
\f1\b0  [\cf2 \outl0\strokewidth0 Asim ur Rahman, Vincenzo Valentino,\super  \cf0 \cb3 \nosupersub Jos\'e9 F. Cobo-D\'edaz, \cb1 Avelino \'c1lvarez Ord\'f3\'f1ez, \cf2 Danilo Ercolini,\super  \nosupersub Francesca De Filippis\cf0 \outl0\strokewidth0 \strokec2 ]. (2025). Custom Meat Spoilage Gene Database. \
\ls6\ilvl0
\f0\b \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	2	}\expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 Associated Manuscript (Please check for updates):
\f1\b0  [\cf2 \outl0\strokewidth0 Asim ur Rahman, Vincenzo Valentino,\super  \cf0 \cb3 \nosupersub \outl0\strokewidth0 \strokec2 Jos\'e9 F. Cobo-D\'edaz, \cb1 \strokec2 Avelino \'c1lvarez Ord\'f3\'f1ez, \cf2 \outl0\strokewidth0 Danilo Ercolini,\super  \nosupersub Francesca De Filippis\cf0 \outl0\strokewidth0 \strokec2 ] (2025). [\cf2 \outl0\strokewidth0 Season, Sanitation, and Bacteriophages Govern Microbial Ecology Across Beef Processing and Maturation Stages\cf0 \outl0\strokewidth0 \strokec2 ]. (Manuscript submitted for publication).\
\pard\pardeftab720\sa240\partightenfactor0
\cf0 \
\pard\pardeftab720\sa298\partightenfactor0

\f0\b \cf0 # Key References for Gene Curation
\f1\b0 \
\pard\pardeftab720\sa240\partightenfactor0
\cf0 The core set of 336 genes was curated based on information from several key publications, including:\
\pard\tx220\tx720\pardeftab720\li720\fi-720\sa240\partightenfactor0
\ls7\ilvl0\cf4 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}{\field{\*\fldinst{HYPERLINK "https://doi.org/10.1016/j.ijfoodmicro.2021.109232"}}{\fldrslt \expnd0\expndtw0\kerning0
\ul \outl0\strokewidth0 \strokec4 https://doi.org/10.1016/j.ijfoodmicro.2021.109232}}\cf0 \expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 \
\ls7\ilvl0\cf4 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}{\field{\*\fldinst{HYPERLINK "https://doi.org/10.1016/j.ijfoodmicro.2024.110675"}}{\fldrslt \expnd0\expndtw0\kerning0
\ul \outl0\strokewidth0 \strokec4 https://doi.org/10.1016/j.ijfoodmicro.2024.110675}}\cf0 \expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 \
\ls7\ilvl0\cf4 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}{\field{\*\fldinst{HYPERLINK "https://doi.org/10.1016/j.foodchem.2024.139045"}}{\fldrslt \expnd0\expndtw0\kerning0
\ul \outl0\strokewidth0 \strokec4 https://doi.org/10.1016/j.foodchem.2024.139045}}\cf0 \expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 \
\ls7\ilvl0\cf4 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}{\field{\*\fldinst{HYPERLINK "https://doi.org/10.1016/j.fm.2024.104466"}}{\fldrslt \expnd0\expndtw0\kerning0
\ul \outl0\strokewidth0 \strokec4 https://doi.org/10.1016/j.fm.2024.104466}}\cf0 \expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 \
\ls7\ilvl0\cf4 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}{\field{\*\fldinst{HYPERLINK "https://doi.org/10.1016/j.tifs.2021.11.007"}}{\fldrslt \expnd0\expndtw0\kerning0
\ul \outl0\strokewidth0 \strokec4 https://doi.org/10.1016/j.tifs.2021.11.007}}\cf0 \expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 \
\ls7\ilvl0\cf4 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}{\field{\*\fldinst{HYPERLINK "http://doi.org/10.26599/FSHW.2022.9250118"}}{\fldrslt \expnd0\expndtw0\kerning0
\ul \outl0\strokewidth0 \strokec4 http://doi.org/10.26599/FSHW.2022.9250118}}\cf0 \expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2  .\
\ls7\ilvl0\cf4 \kerning1\expnd0\expndtw0 \outl0\strokewidth0 {\listtext	\uc0\u8226 	}{\field{\*\fldinst{HYPERLINK "http://dx.doi.org/10.1080/08905436.2014.963601"}}{\fldrslt \expnd0\expndtw0\kerning0
\ul \outl0\strokewidth0 \strokec4 http://dx.doi.org/10.1080/08905436.2014.963601}}\cf0 \expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 \
\pard\pardeftab720\sa240\partightenfactor0
\cf0 \
\pard\pardeftab720\sa298\partightenfactor0

\f0\b \cf0 # Contact
\f1\b0 \
\pard\pardeftab720\sa240\partightenfactor0
\cf0 For questions regarding this database, please contact: Asim ur Rahman ([asimur.rahman@unina.it] or [arahman@bs.qau.edu.pk])\
}